package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "application-manager-server",
-  "version": "0.2.0-3",
+  "version": "0.2.0-7",
```

Comment:
Sorry, I should have just used branches rather than tags to make these builds, will slow down the version bumping in future.